### PR TITLE
Add instructions on how to squash multiple commits

### PR DESCRIPTION
**Pull request by jeffaco**
@yakman2020 @Microsoft/ostc-devs

### CODE REVIEW

**Comment by yakman2020** [comment]
Look good.

**Review by andyleejordan — COMMENTED** [review]
If you already have the commits, just use an interactive rebase to do this automatically. No need to go through the hoops of resetting and committing again.

**Comment by andyleejordan** [comment]
Here's an [explanation](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html). It's way simpler than this proposed way.

File: workflow-workflow.md [hunk 1]
```diff
@@ -374,6 +378,116 @@ Nothing to commit, working directory clean
...
+# Your branch and 'origin/v-brucc-master-pbuild' have diverged,
+# and have 1 and 7 different commits each, respectively.
+#   (use "git pull" to merge the remote branch into yours)
+#
+nothing to commit, working directory clean
+```
+
+Because the `v-brucc-master-pbuild` branch is now divergent from
+the server branch (as indicated in the above status output), the
+branch be force-pushed to the server:
```
Comment (niroyb) [inline workflow-workflow.md]: *must be

File: workflow-workflow.md [hunk 2]
```diff
@@ -374,6 +378,116 @@ Nothing to commit, working directory clean
...
+```
+git commit -m "Changes to build & run regress on AIX, HP, and Solaris platforms" \
+           -m "These changes are on behalf of Bruce Campbell (yackman2020)"
+```
+
+If we now do a `git log`, we see output such as:
+
+```
+commit 3d49ff1 (HEAD, v-brucc-master-pbuild)
+Author: Jeff Coffler <jeff@taltos.com>
```
Comment (niroyb) [inline workflow-workflow.md]: I believe the interactive rebase preserves the author of the commit.

**Review by niroyb — APPROVED** [review]

**Comment by jeffaco** [comment]
Reworked documentation for interactive rebase.

@Microsoft/ostc-devs @andschwa @yakman2020 @niroyb

**Review by andyleejordan — APPROVED** [review]
Looks great!

**Pull request merged**